integration/operate-first/README.md

```diff
@@ -0,0 +1,92 @@
+# Deploying Fybrik on Operate First
+[Operate First](https://www.operate-first.cloud/) is a concept of bringing pre-release open source software to a production cloud environment. The Mass Open Cloud (MOC) is a production cloud resource where projects are run. Deploying Fybrik on Operate First is the first step to integrating Fybrik with Open Data Hub, making Fybrik more easily accessible to data scientists. For further questions about Operate First or contributing to [Operate First GitHub](https://github.com/operate-first) repositories, join the Slack channel [here](http://operatefirst.slack.com/)
+
+## Accessing the [MOC](https://massopen.cloud/) Smaug cluster
+The Smaug cluster is where all user workloads are deployed. A deployment of [Open Data Hub](https://opendatahub.io/) (ODH) is also managed on the Smaug cluster. This is valuable to Fybrik since this ODH Deployment includes JupyterHub.
+
+### Getting Access
+Your GitHub username must be in [this file](https://github.com/operate-first/apps/blob/master/cluster-scope/base/user.openshift.io/groups/fybrik/group.yaml) to get access to the Fybrik user group on the Smaug cluster and login successfully. Create a PR in the [operate-first/apps](https://github.com/operate-first/apps) repository with your GitHub username added to `group.yaml` if you would like to be added as a user. 
+
+### Logging In 
+You can access the Smaug cluster with this [OpenShift console login link](https://oauth-openshift.apps.smaug.na.operate-first.cloud/oauth/authorize?client_id=console&redirect_uri=https%3A%2F%2Fconsole-openshift-console.apps.smaug.na.operate-first.cloud%2Fauth%2Fcallback&response_type=code&scope=user%3Afull&state=98ae2ceb). Click on `operate-first` to login with GitHub authentication. Once logged into the OpenShift console, you can use [this link](https://oauth-openshift.apps.smaug.na.operate-first.cloud/oauth/token/display) to get an `oc login` command with a token that will let you login to the Smaug OpenShift cluster from your terminal. 
+
+## Deploying Fybrik Cluster-Scoped Resources
+This `integration/operate-first` directory contains the raw YAML files of the cluster scoped resources deployed for Fybrik. These files are generated from the Helm charts in `charts/fybrik`.
+
```

Comment:
@eletonia maybe you can add an explanation why the application deployed should not be in cluster scope. is that operate-first requirement? Thanks.

Reply:
Done